pyWAPOR/ETLook/clear_sky_radiation.py

```diff
@@ -327,7 +327,8 @@ def rayleigh_optical_thickness(m):
     """
 
     rotm = 1 / (10.4 + 0.718 * m)
-    rotm[m <= 20] = 1/(6.6296 + 1.7513*m - 0.1202*m**2 + 0.0065*m**3 - 0.00013*m**4)
+    small_m = 1/(6.6296 + 1.7513*m - 0.1202*m**2 + 0.0065*m**3 - 0.00013*m**4)
+    rotm[m <= 20] = small_m[m <= 20]
```

Comment:
Maybe it would be more consistent with the rest of the code to write something like this:
```
rotm = np.where(m <= 20, 1/(6.6296 + 1.7513*m[i] - 0.1202*m[i]**2 + 0.0065*m[i]**3 - 0.00013*m[i]**4), rotm)
```